stream.go

```diff
@@ -283,7 +284,8 @@ type serverStream struct {
 	statusCode codes.Code
 	statusDesc string
 
-	tracing bool // set to EnableTracing when the serverStream is created.
+	tracing bool            // set to EnableTracing when the serverStream is created.
+	ctx     context.Context // provides trace.FromContext when tracing
```

Comment:
this grouping is confusing. at a glance it looks like this is only set when tracing, but that's not the case. I'd put this field higher up in this struct.